src/crate/theme/rtd/crate/static/css/custom.css

```diff
@@ -8,6 +8,11 @@
   margin-bottom: 64px;
 }
 
+a.copybtn {
```

Comment:
can we add `top: .5em;` to that selector so the icon has the same space to top as it does to the right? ;)

Reply:
Sure. I just amended this patch accordingly.